app/views/navbar.scala.html

```diff
@@ -88,6 +88,71 @@
         <!-- /.container -->
 </nav>
     <!-- /.navbar -->
+
+<!-- Script loads regardless of whether user is registered or not -->
+<script>
+$(document).ready(function () {
+    function logWebpageActivity(activity){
+        var currentRoute = window.location.pathname;
+        var url = "/userapi/logWebpageActivity";
+        var async = true;
+        var activityToSend = activity + "_location=Navbar_route=" + currentRoute;
+        $.ajax({
+            async: async,
+            contentType: 'application/json; charset=utf-8',
+            url: url,
+            type: 'post',
+            data: JSON.stringify(activityToSend),
+            dataType: 'json',
+            success: function(result){},
+            error: function (result) {
+                console.error(result);
+            }
+        });
+    }
+
+    // Triggered when 'Sign In' in navbar is clicked
+    // Logs "Click_module=SignIn"
```

Comment:
Can you update all these comments to reflect the changes you made in the code? Like this now logs Click_module=SignIn_location=Navbar_route=<route>